### PR TITLE
Help monitoring impact of community management by Lex

### PR DESCRIPTION
**Pull request by ChristianJoe**

### CODE REVIEW

File: entrypoint/fb.js [hunk 1]
```diff
@@ -165,27 +166,37 @@ const handleTextMessage = async (chat) => {
...
-        case 'CustomPayload':
-            textReply = lexResponse.message;
+    if (chat.trackingEnabled) {
+        await chat.track.event('Conversation-ShortMessage', lexResponse.intentName, chat.language).send();
     }
 
-    if (chat.trackingEnabled) {
-        await chat.track.event('Conversation', lexResponse.intentName, chat.language).send();
+    let textReply;
+    if (lexResponse.message === null) {
```
Comment (jh0ker) [inline entrypoint/fb.js]: Did you test this? If you expect there is no message at all, you should compare to `undefined`

```$ node
> const foo = {}
undefined
> foo.message === null
false
> foo.message === undefined
true
```

File: entrypoint/fb.js [hunk 2]
```diff
@@ -165,27 +166,37 @@ const handleTextMessage = async (chat) => {
...
+        return;
+    } else {
```
Comment (jh0ker) [inline entrypoint/fb.js]: no need for an `else` when we have an early return

File: entrypoint/fb.js (hunk 1 above)
Comment (jh0ker) [reply entrypoint/fb.js]: sorry, it is indeed set to `null` explicitly by the library

File: entrypoint/fb.js (hunk 2 above)
Comment (ChristianJoe) [reply entrypoint/fb.js]: So true. Changed 👌

**Comment by jh0ker** [comment]
thanks, now the diff looks cleaner as well :) LGTM :+1:

**Pull request merged**